src/main/fc/core.c

```diff
@@ -51,6 +51,7 @@
 #include "sensors/boardalignment.h"
 #include "sensors/gyro.h"
 #include "sensors/sensors.h"
+#include "sensors/gyroanalyse.h"
```

Comment:
SITL target is failing because "arm math" is not available on this platform.

Need to wrap in:
`#ifndef SIMULATOR_BUILD`